plugins/woocommerce/tests/e2e-pw/tests/merchant/command-palette.spec.js

```diff
@@ -1,6 +1,10 @@
 const { test, expect } = require( '@playwright/test' );
 const wcApi = require( '@woocommerce/woocommerce-rest-api' ).default;
 
+// need to figure out whether tests are being run on a mac
```

Comment:
These tests fail consistently when running locally, but always pass on CI -- it's because the key combination on Mac is Command+K, not Control+K like it is on Linux.